ajax/libs/expect.js/package.json

```diff
@@ -1,9 +1,26 @@
 {
   "name": "expect.js",
-  "filename": "expect.min.js",
-  "version": "0.2.0",
+  "version": "0.3.1",
+  "license": "MIT",
   "description": "BDD style assertions for node and the browser.",
+  "filename": "index.js",
```

Comment:
`filename` need to be pointed to a minified file. Please also add the minified file of `index.js`. You can use https://github.com/PeterDaveHello/web-minify-helper, or something famous js compress lib.